### ISSUE TITLE
Usability of control used to apply an operation

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/number-line-operations#42]
Noted during #35 code review.

Being the author of Equality Explorer, I had expectations about how this control works:

<img width="285" alt="screenshot_789" src="https://user-images.githubusercontent.com/3046552/103158123-b2b7c380-4777-11eb-8f9b-5832a258d772.png">  

Students who are familiar with Equality Explorer might have similar expectations, so I'll note what I observed here.

(1) I couldn't figure out why I couldn't use the control multiple times. Apparently you can set the operator and operand once, and then the "apply" button is disabled until you reset or erase.   This is very different than Equality Explorer, where you can keep applying operations using the same control.

(2) There are only 2 instances of this control in the carousel, so you're limited to applying 2 operations.  I never figured out why that's a useful limit, and I kept wanting to apply more operations.

(3) Having played with the sim for 45 minutes, I finally discovered (accidentally) that moving a point on the number line changes the operator and/or operand on the control - and scrolls the carousel to show the control that was used to create the point.  There sure is a lot going on here - when I did more the point a whole bunch of stuff happened, and it took me a minute to figure out that the carousel had scrolled _AND_ the operator had changed _AND_ the operand had changed.  How important is it for the student to discover this, and figure out what's going on?

**Comment by pixelzoom** [comment]
(4) I also finally realized that the points (and their labels) that are added get progressively lighter in color.  It's very subtle, and there's no color-coding to the operation control instances. Why not match the picker colors, instead of using the default 'black'?  And will this be a usability problem for users with vision impairments?

**Comment by amanda-phet** [comment]
(1) We want users to be able to edit the operation. If the same control can be used multiple times, they would only be able to edit it using the point controllers. In interviews, this interaction worked well. 

(2) This design is related to the fact that we can't put an unlimited number of operations on the number line. Two is adequate for this grade level, as well, since this is early on in the understanding of operations. 

(3) When you move a point, only the operand should change. If the operator is changing, that is a bug! The auto-scroll feature of the operation control was added so that all of the linked representations were visible at the same time. 

(4) The colors were tested for contrast and color-blindness, so I think they should be fine. I'm fine with making the picker colors coordinate with the points, but we changed the carousel to be color-coordinated too, and that might be too much color. Happy to try this out though. 

@pixelzoom assigning you in case you want to discuss any of these points further. Also tagging this for design meeting to discuss (4).
